src/clr-addons/quick-list/add-option.component.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'add-option',
+  templateUrl: './add-option.component.html',
+  styleUrls: ['./add-option.component.scss'],
+})
+export class AddOptionComponent implements OnInit {
+  @Input() defaultValue: any;
+  @Input() listOfValues: Array<any>;
+  @Input() placeholder: string;
+  @Input() required = false;
+  @Input() displayField: string;
+  @Output() selectedValueChanged = new EventEmitter();
+  @Output() delete = new EventEmitter();
+  selectedValue: any;
+
+  constructor() {}
+
+  ngOnInit() {
+    if (!!this.defaultValue) {
```

Comment:
please also consider the case where you already have values from backend which you want to show